src/NServiceBus.Core/DelayedDelivery/TimeoutManager/ExpiredTimeoutsPoller.cs

```diff
@@ -84,7 +84,7 @@ public void Dispose()
                 if (nextRetrieval > DateTime.UtcNow)
                 {
                     // ReSharper disable once MethodSupportsCancellation
-                    await Task.Delay(1000);
+                    await Task.Delay(1000).ConfigureAwait(false);
```

Comment:
false by default would require app and web devs to use true all the time. No matter what you're gonna piss off ~50% of devs so they decided to piss off library devs.